### PR TITLE
[FIX] project: traceback after disable timesheet from project

### PR DESCRIPTION
**Pull request by yta-odoo**
Purpose of the commit is to remove traceback while accessing projects  menu
(open project module) after disable timesheet from project or uninstall
hr_timesheet

id:2629395

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/hr_timesheet/__init__.py [hunk 1]
```diff
@@ -32,3 +32,14 @@ def create_internal_project(cr, registry):
     } for task in project_ids.task_ids.filtered(lambda t: t.company_id in admin.employee_ids.company_id)])
 
     _check_exists_collaborators_for_project_sharing(env)
+
+def uninstall_hook(cr, registry):
```
Comment (xavierbol) [inline addons/hr_timesheet/__init__.py:36]: Can you rename this method into `_uninstall_hook` ? 🙂

**Review by xavierbol — CHANGES_REQUESTED** [review]
Thanks for your work. 🙂 
I have made a suggestion.

File: addons/hr_timesheet/__init__.py (hunk 1 above)
Comment (mba-odoo) [reply addons/hr_timesheet/__init__.py]: @xavierbol , Done!

**Review by IT-Ideas — APPROVED** [review]
LGTM thanks

**Pull request closed without merging**